source/material_model/modified_tait.cc

```diff
@@ -0,0 +1,288 @@
+          reference_isothermal_bulk_modulus = prm.get_double ("Reference isothermal bulk modulus");
+          reference_Kprime                  = prm.get_double ("Reference bulk modulus derivative");
+          reference_thermal_expansivity     = prm.get_double ("Reference thermal expansivity");
+          einstein_temperature              = prm.get_double ("Einstein temperature");
+
+          eta                               = prm.get_double ("Viscosity");
+          k_value                           = prm.get_double ("Thermal conductivity");
+
+          prm.enter_subsection("Reference heat capacity function");
+          {
+            try
+              {
+                reference_heat_capacity_function.parse_parameters(prm);
+              }
+            catch (...)
+              {
+                std::cerr << "FunctionParser failed to parse\n"
+                          << "\t Reference heat capacity function\n"
+                          << "with expression \n"
+                          << "\t' " << prm.get("Function expression") << "'";
+                throw;
+              }
+          }
+          prm.leave_subsection();
+        }
+        prm.leave_subsection();
+      }
+      prm.leave_subsection();
+
+      // Declare dependencies on solution variables
+      this->model_dependence.thermal_conductivity = NonlinearDependence::none;
+      this->model_dependence.viscosity = NonlinearDependence::none;
+
+
+      this->model_dependence.density = NonlinearDependence::temperature
+                                       | NonlinearDependence::pressure
+                                       | NonlinearDependence::compositional_fields;
+      this->model_dependence.compressibility = NonlinearDependence::temperature
+                                               | NonlinearDependence::pressure
+                                               | NonlinearDependence::compositional_fields;
+      this->model_dependence.specific_heat = NonlinearDependence::temperature
+                                             | NonlinearDependence::pressure
+                                             | NonlinearDependence::compositional_fields;
+
+    }
+  }
+}
+
+// explicit instantiations
+namespace aspect
+{
+  namespace MaterialModel
+  {
+    ASPECT_REGISTER_MATERIAL_MODEL(ModifiedTait,
+                                   "modified tait",
+                                   "A material model that implements the thermal modified Tait "
+                                   "equation of state as written in the paper of "
+                                   "Holland and Powell, 2011 "
+                                   "(``An improved and extended internally consistent "
+                                   "thermodynamic dataset for phases of petrological interest, "
+                                   "involving a new equation of state for solids'', "
+                                   "J. metamorphic Geol., 2011, 29, 333-383, "
```

Comment:
Can you put this citation into `doc/manual/manual.bib` and just `\\cite{...}` it here?